### PR TITLE
Issue 4169 - UI Migrate checkbox to PF4

### PR DESCRIPTION
**Pull request by jchapma**
Description: 	Migrate checkbox from pf3 to pf4
		Button migrations missed in previous PR
                lib/database/ not included in this PR

realtes: https://github.com/389ds/389-ds-base/issues/4169

Reviewed by: ??

### CODE REVIEW

File: src/cockpit/389-console/src/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (jchapma) [inline src/cockpit/389-console/src/.vscode/launch.json]: You can ignore this file, I committed it by mistake.

**Review by droideck — CHANGES_REQUESTED** [review]
A small alignment issue I found:

iss0

**Comment by droideck** [comment]
Looks good!

Could you please also rebase onto master so we can check that everything is okay
(Because there was an RFE with checkboxes in the advanced Database settings)

**Comment by jchapma** [comment]
@droideck 

I rebased with master with no issue

**Comment by mreynolds389** [comment]
Typo:

```
+++ b/src/cockpit/389-console/src/dsModals.jsx
@@ -674,15 +674,21 @@ export class CreateInstanceModal extends React.Component {
                         </Col>
                     </FormGroup>
                     <hr />
-                    <FormGroup controlId="createDBCheckbox">
-                        <Col componentClass={ControlLabel} sm={5} title="Confirm password.">
+                    <FormGroup controlId="createDatabase">
+                        <Col
+                            componentClass={ControlLabel}
+                            sm={5}
+                            title="Create Database"
+                        >
+                            Create Databse
+                        </Col>
+                        <Col sm={7}>
```

Should be "Create Database", not "Create Datbase"

**Comment by mreynolds389** [comment]
Besides the typo in the create instance modal the reset looks good to me...

**Review by droideck — APPROVED** [review]
Besides the typo, @mreynolds389 has mentioned - LGTM too!

**Pull request merged**